packages/fxa-payments-server/public/locales/en-US/main.ftl

```diff
@@ -85,7 +85,27 @@ payment-cvc =
   .label = CVC
 payment-zip =
   .label = ZIP code
-payment-confirm = I authorize Mozilla, maker of Firefox products, to charge my payment method <strong>${ $amount } per { $interval }</strong>, according to payment terms, until I cancel my subscription.
+  
+payment-confirm-day = { $intervalCount -> 
```

Comment:
Does fluent automatically handle the singular and plural cases?

Reply:
Yes! The way this is written, it will use the value assigned to $intervalCount to determine which case to use `[one]` being the singular case (when $intervalCount = 1), or the `*[other]` being the plural case. I added tests to verify the syntax.